fragment.go

```diff
@@ -206,10 +206,11 @@ func (f *Fragment) openStorage() error {
 	if err != nil {
 		return err
 	} else if fi.Size() == 0 {
-		if _, err := f.storage.WriteTo(f.file); err != nil {
+		bi := bufio.NewWriter(f.file)
```

Comment:
would it make sense to add the buffered writer inside `f.storage.WriteTo`? ...maybe not, not sure what best practice is here

Reply:
My preference to put the implementation of the interface as high as possible in the stack as possible